docs/src/theme/DocBreadcrumbs/styles.module.css

```
@@ -1,3 +1,20 @@
+.contextualBack {
+  cursor: pointer;
+  font-weight: 500;
+}
+
+.contextualBack:hover {
+  background: var(--ifm-breadcrumb-item-background-active);
+  color: var(--ifm-link-hover-color);
+  text-decoration: none;
+}
+
+@media (min-width: 62.25rem) {
```

Stylelint (17.14.0)

[error] Expected "context" media feature range notation (media-feature-range-notation)

(media-feature-range-notation)